src/GafferSceneUI/SceneView.cpp

```diff
@@ -163,11 +173,22 @@ class SceneView::DrawingMode : public boost::signals::trackable
 			drawingMode->addChild( points );
 			points->addChild( new BoolPlug( "useGLPoints", Plug::In, true ) );
 
+			ValuePlugPtr lights = new ValuePlug( "light" );
+			drawingMode->addChild( lights );
+			lights->addChild( new StringPlug( "drawingMode", Plug::In, "texture" ) );
+
 			updateOpenGLOptions();
+			updateAttributes();
```

Comment:
Can we not just connect the plugs and avoid the need for `updateAttributes()` entirely?

Reply:
What a splendid idea. Updated https://github.com/GafferHQ/gaffer/pull/3519/commits/fe825436d6a8235c58eeddf573308674eafd1a92 and https://github.com/GafferHQ/gaffer/pull/3519/commits/ea16d84c739008cddc3ea272b0e71dee4999a8b3.